### ISSUE TITLE
discrepancy between test acc of paper and provided code

### ISSUE THREAD

**Issue by mehrnia** [Omid-Nejati/MedViT#25]
Hi, 
Thanks for the great work.
You reported "test  auc: 0.879  acc:0.853" on https://github.com/Omid-Nejati/MedViT/blob/main/Colab_MedViT.ipynb.
However the accuracy of Breast MNIST on paper for tiny medVit is "0.896".
how it's different?

**Comment by mehrnia** [comment]
I just want to bring an attention that. when trained the resnet18 wit same parameters that was mentioned in the paper, I get different ACC and AUC.
Can anyone give me an insight into what approach I should follow?
I have attached the results of 10 independent runs as well.

Image
code:
data_flag = 'breastmnist'
download = True
NUM_EPOCHS = 100
BATCH_SIZE = 128
lr = 0.001
im_size=224
from torchvision.models import resnet18
import timm
device='cuda'
model = resnet18(num_classes=n_classes).cuda()
criterion = nn.CrossEntropyLoss()
optimizer = optim.Adam(model.parameters(), lr=lr)
scheduler = optim.lr_scheduler.MultiStepLR(optimizer, milestones=[50, 75], gamma=0.1)

The result for im_size=224 is also different.
